noir-projects/noir-protocol-circuits/crates/blob/src/utils/compress_to_blob_commitment.nr

```diff
@@ -90,6 +90,25 @@ mod tests {
         assert_eq(reconstructed_y, point.y);
     }
 
+    #[test]
+    unconstrained fn infinity_point_compression() {
+        let point = BLSPoint::point_at_infinity();
+        let (flags, x) = get_flags(point);
+        // is_compressed = true
+        assert_eq(flags[0], true);
+        // is_infinity = true
+        assert_eq(flags[1], true);
+        // is_greater = false (is infinity)
+        assert_eq(flags[2], false);
+
+        assert_eq(x, BLS12_381_Fq::zero());
+
+        // check compression
+        let mut expected_bytes = [0; BLS12_POINT_COMPRESSED_BYTES];
+        expected_bytes[0] = set_flags(flags, 0);
```

Comment:
This line is "marking your own homework", because `compress_to_blob_commitment` also calls `set_flags`.
Consider instead:
```suggestion
        expected_bytes[0] = 0xc0; // 0b11000000
```